internal/common/auth/service_wrapper.go

```diff
@@ -22,6 +22,7 @@ package auth
 
 import (
 	"context"
+	"github.com/golang-jwt/jwt/v5"
```

Comment:
separate std imports from others.

Unrelated @Groxx Should we onboard client to gci too?